src/source/raster_tile_source.js

```diff
@@ -95,11 +118,66 @@ class RasterTileSource extends Evented implements Source {
         this.load();
     }
 
+    reload() {
+        this.cancelTileJSONRequest();
+        this.load(() => this.map.style._clearSource(this.id));
+    }
+
+    setSourceProperty(callback: Function) {
```

Comment:
Interesting. This is a bit of a strange method. The only usage I see is:

https://github.com/mapbox/mapbox-gl-js/blob/3b88dc4e95ed39621fa1adaa4391bc843d98ef33/test/unit/source/vector_tile_source.test.js#L405-L407

So I think it's equivalent to:

```js
source.attribution = 'OpenStreetMap';
source.reload()
```

Maybe it's just a placeholder in case the reload logic gets more complicated. Just jumped out as a bit odd since by the arguments, the method doesn't itself set anything.

Reply:
Hi @rreusser 👋 

Thanks for the review! Previously I used `setSourceProperty` in the VectorTileSource's `setTiles` and `setUrl` methods, so it's more like a leftover. I've removed it 👍

Reply:
Sounds good! At the very least, maybe `wrapSourceMutation(() => {...})` would convey the intent more clearly, but removing works as well. 👍